Dockerfile

```diff
@@ -21,7 +21,7 @@
 #
 # (MIT License)
 
-FROM artifactory.algol60.net/docker.io/alpine:3.13 as service
+FROM artifactory.algol60.net/docker.io/alpine:3.15 as service
```

Comment:
Don't forget to update the copyright date.

Reply:
fixed